myhw-week2/database.js

```diff
@@ -0,0 +1,73 @@
+const mysql = require('mysql');
+const util = require('util');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+
+
+  //Part 1 - 1
+  let country_name = 'Syria';
```

Comment:
The idea of this homework is to take input from the user, not hardcoded in the code :) Consider using `process.argv` for this purpose.